src/main/webapp/app/exam/exam-scores/exam-scores.component.ts

```diff
@@ -70,8 +78,31 @@ export enum MedianType {
     templateUrl: './exam-scores.component.html',
     changeDetection: ChangeDetectionStrategy.OnPush,
     styleUrls: ['./exam-scores.component.scss', '../../shared/chart/vertical-bar-chart.scss'],
+    standalone: true,
+    imports: [
+        FaIconComponent,
+        TranslateDirective,
+        ExportModule,
+        RouterLink,
+        NgStyle,
+        ArtemisParticipantScoresModule,
+        ExamScoresAverageScoresGraphComponent,
+        ArtemisSharedComponentModule,
+        ArtemisSharedCommonModule,
+    ],
```

Comment:
Might it be possible to remove some of those imports? I know it's still difficult, as some important dependencies themselves are not standalone, but have you tried here?

Reply:
thanks for pointing it out, I managed to remove a few imports 👍